### PR TITLE
reworked repo to allow for mix of specializer controller and regular …

### PR DESCRIPTION
**Pull request by henderiw**
I reworked the repo a bit to make it easier to contribute controllers and specializers

### CODE REVIEW

File: controllers/pkg/specializerreconciler/condition.go [hunk 1]
```diff
@@ -14,7 +14,7 @@
  limitations under the License.
 */
 
-package reconciler
+package specializerreconciler
```
Comment (gvbalaji) [inline controllers/pkg/specializerreconciler/condition.go:17]: NIt. should we rename it to specializer-reconciler?

**Review by gvbalaji — APPROVED** [review]
Added a comment on renaming a folder ( minor). Else looks good. 
/approve

File: controllers/pkg/specializerreconciler/condition.go (hunk 1 above)
Comment (henderiw) [reply controllers/pkg/specializerreconciler/condition.go]: done

**Review by vjayaramrh — APPROVED** [review]

**Pull request merged**